pkg/package/manifest.go

```diff
@@ -210,6 +232,10 @@ func parseVariableSchema(manifest []byte) map[string]Schema {
 			if description, okk := vv["description"].(string); okk {
 				schema.Description = description
 			}
+
+			if dflt, okk := vv["default"]; okk {
+				schema.Default = fmt.Sprintf("%v", dflt)
```

Comment:
why the sprintf?

Reply:
`default: 3` in the manifest will have type int, which will fail the the string cast. I was using Sprintf to not have to worry about manually converting each type, but now I'm realizing that maps and slices also won't work. Thoughts?

Reply:
you could do a json marshall and trim off the quotes as a prefix, That way other types will be valid json and strings will be strings